### ISSUE TITLE
CI error: Another metric with the same name already exists in Keras 2.7.0

### ISSUE THREAD

**Issue by albertvillanova** [huggingface/datasets#3207]
## Describe the bug
Release of TensorFlow 2.7.0 contains an incompatibility with Keras. See:
- keras-team/keras#15579

This breaks our CI test suite: https://app.circleci.com/pipelines/github/huggingface/datasets/8493/workflows/055c7ae2-43bc-49b4-9f11-8fc71f35a25c/jobs/52363
